### ISSUE TITLE
ambiguity 6.3 MQTT-Topic Levels /majorVersion

### ISSUE THREAD

**Issue by E-COBOT-AAL** [VDA5050/VDA5050#272]
Hi everyone,

The majorVersion parameter is in the example set as v2 in Example: uagv/v2/KIT/0001/order
This example is the same for implementation 1.1.0 and 2.0.0.

Is this parameter related to the VDA implementation (v1 for implementation 1.1.0 and v2 for 2.0.0) ?

Thanks for your help
Best Regards,

**Comment by AntonDueck** [comment]
yes it do

as written in V1.1
![image](https://github.com/VDA5050/VDA5050/assets/86237773/cb372e70-6fdb-4554-9d09-710d1d353748)
and V2.0
![image](https://github.com/VDA5050/VDA5050/assets/86237773/aa2239b9-6a0e-4a4c-821c-0efa4310eb7b)

**Comment by E-COBOT-AAL** [comment]
Thanks Anton for your answer.

I think that the example in the V1.1 version document is misleading.
The example shows UAGV/V2 where it should always be V1 then because it is describing the major version 1.X.

This has already been misunderstood because other companies we are working with considered that the major version is related to the interface name. And this interfaceName is according to the AGV provider and not related to the standard.

So the expected path on their side is using V1.1 version of standard but the path is interfaceName/**V2**/RobotCompany/0001
And the version of the standard is correctly stated (1.1.0) in the message Header.
According to your reply it should be interfaceName/**V1**/RobotCompany/0001 because they are using version 1.1 of the standard.

Thanks again.
Have a nice day.

**Comment by AntonDueck** [comment]
yes it schuld be the V1 for all V1.x.x versions of the VDA5050. in the ducument  version 1.0.0 is it write the right way.

**Comment by E-COBOT-AAL** [comment]
Would it be possible to request an update of the documents ?
Instead of "Major Version number, preceded by "v"" could it be "Major Version number of the standard, preceded by "v"" ?

This would simplify discussions between AGV navigation system provider and Fleet Management system provider.

**Comment by ConstantinUp** [comment]
This is done in 3.0, we cannot change previously released versions.
Please also check the rework of the recommended structure in PR #546.
